src/app.jsx

```diff
@@ -79,14 +97,18 @@ const NavigatorBreadcrumbs = ({ path, setPath }) => {
                     </Button>
                 </FlexItem>
                 <FlexItem>
-                    <Breadcrumb>
-                        <BreadcrumbItem to="#" onClick={() => setPath([])}>{_("home")}</BreadcrumbItem>
-                        {path.map((dir, i) => {
+                    <Flex>
```

Comment:
It's easy to fix the spacing,
`<Flex spaceItems={{ default: "spaceItemsXs" }}>` will decrease it.